src/v1/texture.rs

```diff
@@ -15,64 +15,64 @@ enum_number!(
         NearestMipmapLinear = 9986,
         LinearMipmapLinear = 9987,
     }
-);
+}
 
 impl Default for Filter {
     fn default() -> Filter {
         Filter::Nearest
     }
 }
 
-enum_number!(
+enum_number! {
     Wrap {
         Repeat = 10497,
         ClampToEdge = 33071,
         MirroredRepeat = 33648,
     }
-);
+}
 
 impl Default for Wrap {
     fn default() -> Wrap {
         Wrap::Repeat
     }
 }
 
-enum_number!(
+enum_number! {
     Format {
         Alpha = 6406,
         Rgb = 6407,
         Rgba = 6408,
         Luminance = 6409,
         LuminanceAlpha = 6410,
     }
-);
+}
 
 impl Default for Format {
     fn default() -> Format {
         Format::Rgba
     }
 }
 
-enum_number!(
+enum_number! {
     Target {
         Texture2d = 3553,
     }
-);
+}
 
 impl Default for Target {
     fn default() -> Target {
         Target::Texture2d
     }
 }
 
-enum_number!(
+enum_number! {
     TexelType {
```

Comment:
I think the following is more terse, self-documenting, and 'rustic':
```rust
TexelType {
    U8 = 5121,
    U16_R5_G6_B5 = 33635,
    U16_R4_G4_B4_A4 = 32819,
    U16_R5_G5_B5_A1 = 32820,
}
```
Let's consider changing this in the next release.